operations/configuration_management/Troubleshoot_CFS_Sessions_Failing_to_Start.md

```diff
@@ -0,0 +1,81 @@
+# Troubleshoot CFS Sessions Failing to Start
+
+Troubleshoot issues where Configuration Framework Service \(CFS\) sessions are being created, but the pods are never created and never run.
+
+## Prerequisites
+
+CFS-batcher is creating automatic sessions, but pods aren't starting for those sessions.  There are a number of communication reasons this could be happening, so check the `cfs-batcher` and `cfs-operator` logs for these signatures.
+
+CFS-batcher logs should show that it is creating sessions, but giving up waiting for those sessions to start:
+
+```bash
+2022-08-24 04:09:12,391 - WARNING - batcher.batch - Session batcher-d7f9bf52-e5f6-4742-849b-4086b1d59ab1 is stuck in pending and will be deleted.
+2022-08-24 04:09:12,492 - WARNING - batcher.batch - Session batcher-77c3c4d1-df39-4ebc-a6b4-32b8e102bdc5 is stuck in pending and will be deleted.
+2022-08-24 04:09:12,638 - WARNING - batcher.batch - Session batcher-f9a513c1-47ad-4539-98cc-b3f418f6b8bd is stuck in pending and will be deleted.
+2022-08-24 04:09:12,761 - WARNING - batcher.batch - Session batcher-7a291f2f-0149-46d7-889b-08331a46af2c is stuck in pending and will be deleted.
+2022-08-24 04:09:15,144 - WARNING - batcher.batch - Session batcher-b6c35dcc-b8b9-421b-9090-23bfc2a72ac3 is stuck in pending and will be deleted.
+```
+
+CFS-operator logs should show that it is attempting to create sessions but is unable to find the session records.
+
+```bash
+2022-08-24 05:48:06,476 - INFO    - cray.cfs.operator.events.session_events - EVENT: CREATE batcher-3b78941e-1c06-474c-89fe-bf241f5002e4
+2022-08-24 05:48:06,509 - ERROR   - cray.cfs.operator.cfs.sessions - Unexpected response from CFS: 404 Client Error: Not Found for url: http://cray-cfs-api/v2/sessions/batcher-3b78941e-1c06-474c-89fe-bf241f5002e4
+```
+
+If these two things are true, it is likely that CFS is creating new sessions faster than it can keep up with session creation events.
```

Comment:
```suggestion
If these two things are true, then it is likely that CFS is creating new sessions faster than it can keep up with session creation events.
```